packages/completer/src/widget.ts

```diff
@@ -1010,7 +1021,8 @@ export namespace Completer {
       const matchNode = document.createElement('code');
       matchNode.className = 'jp-Completer-match';
       // Use innerHTML because search results include <mark> tags.
-      matchNode.innerHTML = defaultSanitizer.sanitize(result, {
+      const sanitizer = new Sanitizer();
+      matchNode.innerHTML = sanitizer.sanitize(result, {
```

Comment:
Can we use the newly added `this.sanitizer` here to avoid creating thousands of instances of Sanitizer?

Reply:
Sure thing on adding a note to the [extension migration guide](https://github.com/jupyterlab/jupyterlab/blob/master/docs/source/extension/extension_migration.rst). I'll try to write something up.

Will change to `this.sanitizer`, it isn't ideal to keep instantiating new sanitizer's. Thanks for the pickup.

Reply:
> Can we use the newly added `this.sanitizer` here to avoid creating thousands of instances of Sanitizer?

`this` in this instance refers to 'IRenderer' which does not have a `sanitizer` attribute. Should I define one there?

Reply:
Yes, please do add it to `IRenderer` as well.